synapsemonitor/actions.py

```diff
@@ -22,3 +23,29 @@ def action(self):
         )
         action_result = self._action(modified_entities)
         return action_result
+
+
+class EmailAction(SynapseAction):
+    """This action emails specified users with modified entities"""
+    def __init__(self, syn: Synapse, syn_id: str, days: int = 1,
+                 users: list = None,
+                 email_subject: str = "New Synapse Files"):
+        self.users = users
+        self.email_subject = email_subject
+        super().__init__(syn=syn, syn_id=syn_id, days=days)
+
+    def _action(self, modified_entities: list) -> list:
```

Comment:
It would be great to have the flexibility for different formats of emails or other variations on an Action class' behavior. Maybe another parameter to _action(self, modified_entities, format) that determines calls to other functions in the Action class?  For example, with the EmailAction, the function to construct the body could be selected via this parameter with a default function for body format being a comma-delimited list.  

if format == 'default':
  body = _default(modified_entities)
elif format == 'fancy':
  body = fancy(modified_entities)

etc...

Reply:
Added to issue #17